### PR TITLE
Add a known issue in the CRI doc

### PR DESCRIPTION
**Pull request by yujuhong**
Also added some more github issue numbers for them.

/cc @Random-Liu @freehan @timstclair

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37331)

### CODE REVIEW

File: docs/devel/container-runtime-interface.md [hunk 1]
```diff
@@ -79,38 +77,44 @@ besides discussions on Github issues.
...
 #### [CRI known issues](#cri-1.5-known-issues):
 
-  - Container metrics are not defined yet in CRI ([#27097](https://github.com/kubernetes/kubernetes/issues/27097)).
-  - CRI may not be compatible with other experimental features (e.g., Seccomp)
-  - Streaming server needs to be further productionized:
-     - Authentication: [#36666](https://github.com/kubernetes/kubernetes/issues/36666)
-     - Avoid including user data in the redirect URL: [#36187](https://github.com/kubernetes/kubernetes/issues/36187)
-
+  - [#27097](https://github.com/kubernetes/kubernetes/issues/27097): Container
+    metrics are note yet defined in CRI.
```
Comment (dims) [inline docs/devel/container-runtime-interface.md]: s/note/not/

**Comment by feiskyer** [comment]
LGTM

**Comment by Random-Liu** [comment]
LGTM

**Comment by yujuhong** [comment]
Applying lgtm based on the comments. @dchen1107 let me know if you want me to change anything.

**Pull request merged**